### PR TITLE
fix(DB/Skinning_loot): Remove Chimaera Leather

### PR DESCRIPTION
**Pull request by syssneck**
Removes Chimaera Leather from `skinning_loot_template` (this item was removed from Skinning Loot in Patch 3.1.0). Furthermore, this PR increases the Skinning Loot chance for all remaining Skinning Loot (Thick Leather, Thick Hide (if present), Rugged Leather & Rugged Hide) to add up to 100 %, so that there is no chance of no Skinning Loot after removing Chimaera Leather.

The increase was calculated by multiplying with the following factor: 100/(100 - [%chance of Chimaera Leather]).
We don't have any blizzlike data for the exact chances without Chimaera Leather after Patch 3.1.0, so this is the best I can do.

## Changes Proposed:
-  Removes Chimaera Leather from `skinning_loot_template`
-  Increases the chance for Thick Leather, Thick Hide (if present), Rugged Leather & Rugged Hide appropriately (sum =100 % -> always skinning loot)

## Issues Addressed:

- Closes https://github.com/azerothcore/azerothcore-wotlk/issues/7377)
- Closes https://github.com/chromiecraft/chromiecraft/issues/1426)

## Tests Performed:

- Ran the SQL, no errors
- Checked the db, no more Chimaera Leather and updated Skinning Loot chances
- no in-game tests so far

## How to Test the Changes:

1. Go to any guid of the following creatures:
grafik
2. Kill the creature
3. Loot the creature
4. Skin the creature
5. Repeat multiple times and for all creatures
6. See that you don't get any Chimaera Leather
7. See that you always get Skinning Loot (never empty)
8. Additionally, check the db with the changes

## Known Issues and TODO List:

- [ ]
- [ ]

## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.

### CODE REVIEW

File: data/sql/updates/pending_db_world/rev_1629290838385347720.sql [hunk 1]
```diff
@@ -0,0 +1,56 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1629290838385347720');
+
+-- Delete Chimaera Leather
+DELETE FROM `skinning_loot_template` WHERE `item` = 15423 AND `entry` IN (7447, 7448, 7449, 8763, 8764, 10807, 11497);
```
Comment (Helias) [inline data/sql/updates/pending_db_world/rev_1629290838385347720.sql:4]: you need only this line to remove the item from the skinning_loot_template, don't need to re-add again all the items in any skin_loot_template, on Keira3 you can use the diff sql script and not the full one

Comment (syssneck) [reply data/sql/updates/pending_db_world/rev_1629290838385347720.sql:4]: I already used the diff script (Keira3 3.2.0). 
Please read the full PR, I think there is a misunderstanding. 
I deleted Chimaera Leather and then changed the loot chance for all the other Skinning Loot (see above).

Comment (Helias) [reply data/sql/updates/pending_db_world/rev_1629290838385347720.sql:4]: you're right, sorry.

**Review by Helias — APPROVED** [review]

**Comment by Helias** [comment]
I would like if anyone test this even in-game at least once

**Comment by Azcobu** [comment]
Ran through and tested this, did all of the listed NPCs at least 10x each except The Razza - porting to it left me in Dire Maul's arena with no Razza in sight, so apparently some event needed to spawn him. Other than that, the only strange thing was finding one Chillwind Chimaera (41253) that was falling through the world, which had nothing to do with this PR - otherwise, all looked good. No weird leather drops found, no Chimaera Leather ever appeared, and no blank/empty skins.

**Comment by FrancescoBorzi** [comment]
good job guys, don't forget [To Be Merged] after removing [Waiting To Be Tested]

**Pull request merged**